build.gradle

```diff
@@ -7,7 +7,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "5.12.0-SNAPSHOT"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "5.13.0-RDM-9554"
```

Comment:
we are doing the pre-release instead of changing things in build.gradle..

Reply:
Did the pre-release to be able to build the data-store PR. Will need to release it under a 5.13.0 version when PR is approved. Then bump in master to 5.14.0-SNAPSHOT.